test/integration/lib/philipsHue_server.js

```diff
@@ -107,14 +108,21 @@ var philipsHue_server = (function() {
         res.status(200).json(response);     
       });
 
-      _server.listen(port, function () {
-        console.log('Hue simulator listening on port ' + port);
-      });
-    }
+    instance = _server.listen(port, function () {
+      console.log('Hue simulator listening on port ' + port);
+    });
+  }
+
+  function stop(callback) {
+    instance.close(function(){
+      console.log('philips hue server closed');
+      return callback;
```

Comment:
so this releases the port.  otherwise, on the next test run, the test will fail because the port is still occupied. I have to make sure that the listener instance is closed, by waiting on the callback